db/seeds.rb

```diff
@@ -5,3 +5,34 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+# # Post
+# Post.create([
+#   {
+#     content: "test post 1", 
+#     user_id: 1
+#   },
+#   {
+#     content: "test post 2", 
+#     user_id: 2
+#   },
+#   {
+#     content: "test post 3", 
+#     user_id: 3
+#   }
+# ])
+
+# # SharedPost
+# SharedPost.create([
+#   {
+#     user: User.first, 
+#     post: Post.second
+#   },
+#   {
+#     user: User.first, 
+#     post: Post.third
+#   },
+#   {
+#     user: User.second, 
+#     post: Post.third
+#   }
+# ])
```

Comment:
If this code is unnecessary please remove it, and try to not push the comment code.

Reply:
Ok, done. 🫡